scripts/kani-regression.sh

```diff
@@ -64,6 +65,11 @@ else
   TESTS+=("no_unsound_experiments expected")
 fi
 
+# Build compiletest and print configuration. We pick suite / mode combo so there's no test.
+echo "--- Compiletest configuration"
+cargo run -p compiletest --quiet -- --suite kani --mode cargo-kani --dry-run --verbose
```

Comment:
It's not clear to me why printing the configuration with an invalid suite/mode pair is useful here.

Reply:
My main motivation to add this was to get more information out of the builders that are being used today. Specially, the number of threads used to run our tests. I thought it might be helpful to keep logging it to understand why some runs takes longer than others. So far, it looks like Mac uses 3 and Ubuntu workers run 2 tests in parallel. 

Not providing a `--suite` and a `--mode` is a hard failure today, and I wanted to limit the size of this change, since I'm really just trying to get some information to help us implement these time out. So I picked a combo that doesn't find any test but that doesn't fail the run.

I also found it very helpful to list the tests being executed, hence, the `--dry-run`. I'm happy to fine tune this or drop it. The timeout part is the most important anyway.